src/main/java/services/VmBackupsService.java

```diff
@@ -142,6 +142,26 @@ default void inputDetail() {
          * @since 4.4.4
          */
         @In Boolean requireConsistency();
+
+        /**
+         * Indicate whether to use the active volume for performing the backup.
+         *
+         * If useActive=False a snapshot will be created for the backup operation.
+         *
+         * The REST API call should look like this:
+         *
+         * ....
+         * POST /ovirt-engine/api/vms/123/backups?use_active=false
+         * ....
+         *
+         * The default value of the useActive flag is `false`.
+         *
+         * @author Benny Zlotnik <bzlotnik@redhat.com>
+         * @date 24 Mar 2022
+         * @status added
+         * @since 4.5
+         */
+        @In Boolean useActive();
```

Comment:
Did you consider naming this property 'useActiveVolume'? 

Would that in your opinion add enough clarity to justify the extra word? 

I always lean towards very meaningful names even if they are a bit longer, but it's a matter of taste

Reply:
In second thought I don't think it makes much of a difference, I will merge this as is